apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Utils.cs

```diff
@@ -14,13 +14,17 @@
 
 using System;
 using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Google.Cloud.Diagnostics.Common.Tests
 {
     internal class Utils
     {
         private const string _projectEnvironmentVariable = "TEST_PROJECT";
 
+        private static readonly Task s_completedTask = Task.FromResult(1);
```

Comment:
Is this needed anymore? It seems like this assembly has internals visible to it from Common. Can you just use `CommonUtils. CompletedTask`?